gui/storage/models.py

```diff
@@ -287,6 +292,11 @@ def delete_attachments(self):
             for device_id in attachments['vm']:
                 Device.objects.filter(id=device_id).delete()
 
+        if attachments['jails_ng']:
+            with client as c:
+                for ioc_j in attachments['jails_ng']:
+                    c.call('jail.delete', ioc_j)
+
```

Comment:
We are not dedleing old jails we should not delete new jails either.

This is simply to stop the service so the pool can be cleanly exported. See how its done for old jails.